secret-store/Cargo.toml

```diff
@@ -42,10 +35,8 @@ jsonrpc-server-utils = "14.0.3"
 
 [dev-dependencies]
 env_logger = "0.5"
-ethkey = { path = "../accounts/ethkey" }
-ethcore = { path = "../ethcore", features = ["test-helpers"] }
 tempdir = "0.3"
 kvdb-rocksdb = "0.1.5"
 
 [features]
-accounts = ["ethcore-accounts"]
+accounts = []
```

Comment:
I think this feature isn't required anymore, right?

Reply:
hmm, how is the `SS RPC interface` suppose to work without accounts? Or is it disabled by default?